Dockerfile

```diff
@@ -0,0 +1,26 @@
+# s2i-custom-notebook
+FROM quay.io/thoth-station/s2i-minimal-notebook:master
```

Comment:
@harshad16 is this the correct base image to use?

Reply:
yes it the correct base image :+1: